Fullscript/fullscript-js
# Tokenize patient data if provided

Closes [Gitlab ticket](https://git.fullscript.io/developers/hw-admin/-/issues/27110)

-- Minor security patch --

### What's changed?
- How the iframe url is being generated: If the patient info is passed to the library, we now make a call to the backend to tokenize this data as opposed to having it exposed.

**Usage of the package from the client-side remains unaffected**

### Testing
1. Run fullscript.js locally, using the demo folder (see read-me for instructions).
2. In `App.tsx`, pass any patient info, like
```
patient: {
    id: "a23a465f-20db-4602-a53b-3f283d90e388",
    email: "mail@google.com",
    firstName: "What name",
}
```
3. Run the demo app, and inspect the iframe url in devtools. The patient info provided shouldn't be visible in the url
4. Select a prac, and then 'Create patient'
5. Verify that the provided patient info is pre-populated in the patient form


https://user-images.githubusercontent.com/22198877/192529447-110af640-cc05-464b-873e-0d3950bbc1de.mov
